### DISCUSSION TITLE
How to join the slack #arrow-rust channel?

### DISCUSSION THREAD

**Discussion by HaoYang670** [apache/arrow-rs#2039] [Q&A] [answered]
Hi all. 
I want to join the slack #arrow-rust channel, but it needs someone to invite me. 
Could you help me?

**Comment by HaoYang670** [comment, marked as answer]
@alamb Just curious that is the Slack channel a main discussion area for arrow-rs? And is it a public channel or an internal channel for committers or maintainers?

**Reply by alamb** [reply]
Hi @HaoYang670  I would not say it is a high volume discussion channel and all the information eventually makes it to the normal dev@arrow.apache.org mailing list or github, but it is a good way to communicate with other contributors. 

For some reason, the ASF slack does not allow anyone to join -- it requires an existing member to send an invite. If you tell me an email address I can send an invite to you.

looking at the docs in https://github.com/apache/arrow-rs#arrow-rust-community this information does not appear to be present. We should probably update it 🤔

**Reply by alamb** [reply]
Filed https://github.com/apache/arrow-rs/issues/2192 to discuss

**Reply by HaoYang670** [reply]
Thank you @alamb for your reply.
I'd like to join the slack channel if it is possible. 
My email address is 13716567376yh@gmail.com. Thank you very much.

**Reply by alamb** [reply]
Invite sent

**Reply by HaoYang670** [reply]
Received. Thank you @alamb
